articles/active-directory/external-identities/user-properties.md

```diff
@@ -83,7 +83,7 @@ Microsoft account |  This user is homed in a Microsoft account and authenticates
 {host’s domain} | This user authenticates by using an Azure AD account that belongs to this organization.
 google.com | This user has a Gmail account and has signed up by using self-service to the other organization.
 facebook.com | This user has a Facebook account and has signed up by using self-service to the other organization.
-mail | This user has an email address that doesn't match with verified Azure AD or SAML/WS-Fed domains, and isn't a Gmail address or a Microsoft account.
+mail | This user has an email address that doesn't match with verified Azure AD or SAML/WS-Fed domains, and has been authenticated using Azure AD Email one-time passcode.
```

Comment:
@ChristianJBergstrom would you please change the description to the following? Then I'll go ahead and sign off. Thanks!
```suggestion
mail | This user has signed up by using Azure AD Email one-time passcode (OTP).
```

Reply:
@msmimart Not sure I did it the correct way, need a training here 😅 Updated now though.